front_end/ndb_ui/NodeModulesBlackboxing.js

```diff
@@ -33,7 +33,7 @@ Ndb.NodeModulesBlackboxing = class extends UI.VBox {
   async _update() {
     const processManager = await Ndb.NodeProcessManager.instance();
     const {error, code, stderror, stdout} = await processManager.run(NdbProcessInfo.npmExecPath, ['ls', '--depth=0', '--json']);
-    if (error || code !== 0 || stderror)
+    if (!stdout || stdout.length === 0)
```

Comment:
Why this change?

Reply:
it is based on comment from the bug: https://github.com/GoogleChromeLabs/ndb/issues/45#issuecomment-408181317
npm run --json sometimes produce pretty valid stdout but dumps something to stderr as well.